docs: initial draft for domain migration (#64)
dwhswenson/playbook-website: content/cloudflare/domain-migration.md

---
layout: topic
title: Migrating Domains to Cloudflare
weight: 3
---
## Concepts

### DNS
DNS or Domain Name System is how domain names get translated into IP addresses. 
When you buy a domain, you buy the ability to set those IP addresses.

> Squarspace, GoDaddy, and Cloudflare are all examples of the registrars 

Reviewer: ```suggestion
Squarespace, GoDaddy, and Cloudflare are all examples of registrars.
```

But is this line necessary, given the next section?

For a technical deep dive, see [this article from Cloudflare](https://www.cloudflare.com/learning/dns/what-is-dns/).

### Registrar
A registrar is who you buy a domain from.

> Some examples of this are Squarspace, GoDaddy, and Cloudflare.

Reviewer: ```suggestion
Some examples of this are Squarespace, GoDaddy, and Cloudflare.
```

They are responsible for your billing.

### Cloudflare Management
Cloudflare supports two approaches to domain configuration: **managed** and **owned**.
Under the *managed* approach, Cloudflare handles all of your DNS records while your domain registration remains with your existing registrar. 
If you purchased your domain through a different registrar, such as Squarespace, you would continue to pay that registrar for the domain itself while managing your DNS records, adding, and updating them as needed, within Cloudflare.
Under the *owned* approach, you migrate your domain registration to Cloudflare directly. 
In this configuration, Cloudflare serves as both your DNS provider and your registrar, meaning Cloudflare handles domain registration and billing.

> This tutorial covers how to configure both approaches.

Reviewer: I'm not sure that this sentence is clearly true with the current text here (not sure that it needs to be true, but the it wouldn't be clear to me what where exactly I look to see each approach). Suggestions to deal with this in the main review comment.

## Management and Migration

### Managing Your Domain with Cloudflare
This process is straightforward and can be found using [Cloudflare's docs](https://developers.cloudflare.com/dns/zone-setups/full-setup/setup/).

### Migrating Your Domain to Cloudflare
This process is documented on [Cloudflare's docs](https://developers.cloudflare.com/registrar/get-started/transfer-domain-to-cloudflare/).
However, it does not cover how you set this up for popular registrars. See below for how to handle this.
Use the following to migrate from common registrars:

> - [Squarspace](https://support.squarespace.com/hc/en-us/articles/205812338-Transferring-a-domain-away-from-Squarespace)

Reviewer: ```suggestion
- [Squarespace](https://support.squarespace.com/hc/en-us/articles/205812338-Transferring-a-domain-away-from-Squarespace)
```
